apps/api/src/app/events/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -41,12 +41,24 @@ export class SendMessageInApp extends SendMessageType {
       _id: command.subscriberId,
     });
     const inAppChannel: NotificationStepEntity = command.step;
-    const content = await this.compileInAppTemplate(
-      inAppChannel.template.content,
-      command.payload,
-      subscriber,
-      command
-    );
+
+    let content: string;
+
+    try {
+      content = await this.compileInAppTemplate(inAppChannel.template.content, command.payload, subscriber, command);
+    } catch (e) {
+      await this.sendErrorStatus(
+        null,
+        'error',
+        'in-app_error_while_compiling_template',
+        e.message,
+        command,
+        notification,
+        LogCodeEnum.TEMPLATE_COMPILE_ERROR
+      );
+
+      throw e;
+    }
```

Comment:
@scopsy let me know if this log needed, as someone who figured out what the problem was after about ten minutes

Reply:
I think we can address this on the next cycle from the root cause to display it better.